apps/voting/.solcover.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   skipFiles: [
     'test',
     '@aragon/os',
-    '@aragon/apps-shared-minime',
+    '@aragon/minime',
```

Comment:
Just to double check, did we update the package.jsons to use `@aragon/minime` too?

Reply:
Yes, this was updated on `package.json` but we missed to update `.solcover.js`.